shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -30,4 +36,87 @@ - (BOOL)enableInputClicksWhenVisible {
   return YES;
 }
 
+void SnapshotRasterizer(ftl::WeakPtr<shell::Rasterizer> rasterizer,
+                        CGContextRef context,
+                        bool is_opaque) {
+  if (!rasterizer) {
```

Comment:
P.S. I could be wrong about this, I can see both styles of code in adjacent files, probably it just depends who wrote it. so feel free to ignore me :)